usage/jsgui/src/main/webapp/assets/tpl/home/applications.html

```diff
@@ -40,8 +40,8 @@
     <div id="applications">
         <table class="table table-striped table-bordered table-condensed">
             <thead>
-            <th>Application</th>
-            <th>Status</th>
+            <th><h3>Application</h3></th>
```

Comment:
The HTML standard forbids the use of Heading Content in a `th` element.  Suggest using css to style them instead.